Yosemite/Yosemite/Actions/SystemStatusAction.swift

```diff
@@ -12,6 +12,10 @@ public enum SystemStatusAction: Action {
     ///
     case fetchSystemPlugin(siteID: Int64, systemPluginName: String, onCompletion: (SystemPlugin?) -> Void)
 
+    /// Fetch an specific systemPlugin by siteID and name list.
+    ///
+    case fetchSystemPluginList(siteID: Int64, systemPluginNameList: [String], onCompletion: (SystemPlugin?) -> Void)
```

Comment:
super nit: the case name sounds like it's fetching a list of plugins, while the completion block returns up to one. If Swift enums don't support the same case name with different associated types, maybe `fetchSystemPluginListWithNameList` 🤔 up to you!

Reply:
Yeah I wasn't happy with the name either!